internal/filesystem.go

```diff
@@ -284,6 +284,9 @@ func FileSha256(filepath string) (sum []byte, err error) {
 // checks if command is in PATH
 func IsCommandAvailable(command string) bool {
 	_, err := exec.LookPath(command)
+	if err != nil {
```

Comment:
I don't think it should print anything to log when a command is unavailable. For example on Debian the last method is used, so it would always print two warnings.

Reply:
Probably you are right, at first I had this logging for my own testing/validation, later I left it thinking maybe it could be useful in real life when investigating end user situations to know which dns handling methods were tried (with what outcome) and which used in fact. To be removed.